input/resources/au-core-responder.xml

```diff
@@ -819,6 +851,49 @@
 </div>
 </div>
 </div>
+<div class="panel panel-default">
+<div class="panel-heading">
+<h4 id="DocumentReference1-1" class="panel-title">
+<span style="float: right;">Resource Conformance: SHOULD</span>DocumentReference</h4>
+</div>
+<div class="panel-body">
+<div class="container">
+<div class="row">
+<div class="col-lg-6">
+<span class="lead">Base System Profile</span>
+<br/>
+<a href="http://hl7.org/fhir/StructureDefinition/DocumentReference">http://hl7.org/fhir/StructureDefinition/DocumentReference</a>
+</div>
+<div class="col-lg-3">
+<span class="lead">Profile Conformance</span>
```

Comment:
The Profile Conformance is SHALL. Apply this here and in the structure.

Reply:
Applied profile conformance to narrative, bolded SHALL in documentation in for narrative and structure.